### ISSUE TITLE
Workspacer seems to disable itself sometimes

### ISSUE THREAD

**Issue by sim590** [rickbutton/workspacer#168]
I don't know how to reproduce it right now, but what I observe is windows suddenly don't minimize themselves when changing workspace. However, windows in the selected workspace come up as expected. I think that is the behaviour that the option "disable workspacer" produces when you press it in the drop menu after right clicking the workspacer icon in the tray. Indeed, I can resolve the issue by clicking the option which is a toggle, hence workspacer is re-enabled.

Version: 0.9.10

**Comment by NeoNlAmp3** [comment]
I can confirm the issue, I use alt + q to restart workspacer.
Also when resetting often the windows can get lost and then need to be killed with the Task Manager.

**Comment by josteink** [comment]
That's 2 reports in short time... after having no such reports for a long time. That's alarming!

Could this be a recent regression? Can any of you confirm if you have this issue or not with version 0.9.9?

**Comment by sim590** [comment]
I will confirm it by the end of the day (if I don't forget about it).

**Comment by sim590** [comment]
I've looked at it and there is no release for that version number. I didn't want to look into how to compile the project, so I didn't confirm yet. Is that just a matter of inserting the solution in Visual Studio? If so, I'll try that.

**Comment by josteink** [comment]
Yeah. It's pretty much just opening and building in Visual Studio.

**Comment by sim590** [comment]
Well, I get an exception while launching the program after building the solution. I don't have the energy to dig into this. If I could have a binary release to test it, I could answer your question. In the meantime, Workspacer is almost unusble I feel...

**Comment by N1x0** [comment]
@sim590  releases are available on github, scoop and chocolatey, so you should be able to get it installed quite easily. If you're still experiencing this issue do let me know and I'll have a look

**Comment by sim590** [comment]
Since my last message, I have found a way to reproduce the issue. For me, I just have to keep <kbd>alt</kbd> pressed and successively press <kbd>esc</kbd> then <kbd>j</kbd>. This will trigger the weird state that I described above where windows from the last visisted workspace will still be displayed and not be minimized or hidden while windows from the current workspace will also show up. More precisely, do the following:

1. Place two windows in workspace 1
2. Place 1 window in workspace 2
3. Hit <kbd>alt</kbd>+<kbd>esc</kbd>, <kbd>alt</kbd>+<kbd>j</kbd>
4. Go to workspace 1, then notice that window from workspace 2 is still displayed.

This state can be reverted by hitting <kbd>alt</kbd>+<kbd>esc</kbd>, <kbd>alt</kbd>+<kbd>j</kbd> again. Then, to make windows disappear, simply visit all the workspaces successively and notice how the windows will fall back in their respective workspace.
